test/test_rosbag/test/test_bag.py

```diff
@@ -312,6 +312,14 @@ def test_get_time(self):
             self.assertEquals(start_stamp, 0.0)
             self.assertEquals(end_stamp, 99.0)
 
+    def test_get_time_empty_bag(self):
+        """Test for issue #657"""
+        fn = '/tmp/test_get_time_emtpy_bag.bag'
```

Comment:
How does this file get put into `/tmp`?

Usually we download test data as part of the test procedure, for example:

https://github.com/ros/ros_comm/blob/indigo-devel/test/test_rosbag/CMakeLists.txt#L18

If you want I can place the test bag file on our servers.